src/lib/telemetry.rs

```diff
@@ -11,17 +13,41 @@ use tracing_subscriber::{layer::SubscriberExt, EnvFilter, Registry};
 use crate::settings::Settings;
 use crate::version::{read_version, VERSION_FILE};
 
+#[derive(Debug)]
+pub enum TraceType {
+    AicRecordCreate,
+    AicRecordCreateFailed,
+}
+
+impl fmt::Display for TraceType {
```

Comment:
An alternative to this could be to derive `strum_macros::Display` and then applying a `serialize` attribute to each enum value.

https://docs.rs/strum_macros/0.24.0/strum_macros/derive.Display.html

Reply:
Updating.